docs/cep/query-guide/functions/sink/kafka.md

```diff
@@ -45,4 +46,4 @@ This Kafka sink configuration publishes to 0th partition of the topic named `top
     insert into BarStream
```

Comment:
`@info(name = 'query1)` makes more sense to go above the query than above sink creation.
Also, map.type should be `json` instead of `xml`, since we don't currently support `xml`.
This should be changed in both examples.

Reply:
Please either make a commit on this branch PR or make the changes in suggestions.

Reply:
@StoyanTc, can you help me understand what he means here? Or just make the changes yourself, if that is faster?

Reply:
@diana-macrometa changes have already been made in separate PR, which you have merged earlier, so this comment can be dismissed